data/brands/amenity/bicycle_rental.json

```diff
@@ -1644,14 +1644,15 @@
     },
     {
       "displayName": "Hamilton Bike Share",
-      "id": "hamiltonbikeshareinc-3eadaa",
+      "id": "hamiltonbikeshare-3eadaa",
       "locationSet": {"include": ["ca"]},
```

Comment:
notice：
you don't need to change id because operator have the word Inc so every time program will use operator generate it

Reply:
Didn't notice that. I ran npm before committing so probably it was done automatically :)